packages/vuepress-theme-odyssey/styles/docskit/_Visual.scss

```diff
@@ -82,6 +82,16 @@
     }
   }
 
+  &.is-docskit-visual-content-no-padding {
+    .docskit-visual--figure {
+      grid-auto-columns: 1fr;
+      padding-block-start: 0;
+      padding-block-end: 0;
+      padding-inline-start: 0;
+      padding-inline-end: 0;
+    }
+  }
```

Comment:
I realize it isn't a huge amount of effort to write a few lines of CSS for the vue-press theme but if we never plan to merge this work why spend the effort formatting this to be displayed via the hosted site at all? I feel like we could streamline this type of RFC process by having this writeup live someplace else where we are free from these types of non critical concerns?

Reply:
I'm not sure why we wouldn't ship this work or update the documentation to reflect the updated component breakdown. Once the spec and styling are finalized, these doc changes will be necessary to display fully assembled forms as we expect them to render in product UI.